inspirehep/config.py

```diff
@@ -719,3 +719,10 @@
 RT_PASSWORD = None
 BIBCATALOG_QUEUES = "Test"
 JLAB_ARXIV_CATEGORIES = []
+
+
+# URLS
+UI_SERVER_NAME = "localhost:8081"
+INSPIREHEP_URL = f"https://{UI_SERVER_NAME}"
+JOB_DETAILS = f"{UI_INSPIREHEP_URL}/jobs/<pid_value>"
+JOB_EDIT = f"{UI_INSPIREHEP_URL}/submissions/job/<pid_value>"
```

Comment:
I would use string format instead of replacing <pid_value>

Reply:
I also thought about this, but we are not using format anymore, am I right?